dnf-plugins-core.spec

```diff
@@ -159,7 +159,9 @@ Additionally provides generate_completion_cache passive plugin.
 
 %if %{with dnfutils}
 %package -n dnf-utils
-Conflicts:      yum-utils < 1.1.31-513
+# we need to specify the version since we would conflict with ourselves
+# otherwise (see the Provides below)
+Conflicts:      yum-utils <= 1.1.31-519
```

Comment:
Could be simpler to 
Conflicts: yum-utils < 1.1.32

Reply:
If retired from F31, rather

```
%if 0%{?fedora} >= 31
Obsoletes: yum-utils < 1.1.32
Provides: yum-utils = %{version}
%else
Conflicts: yum-utils < 1.1.32
%endif
```

Reply:
https://docs.fedoraproject.org/en-US/packaging-guidelines/#renaming-or-replacing-existing-packages

The guidelines suggest just bumping the release one higher, not the version. 
 I think `yum-utils < 1.1.31-520` is the right choice.